### PR TITLE
Migrate all lovelace elements to mwc

### PR DESCRIPTION
**Pull request by bramkragten**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [X] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/lovelace/cards/hui-shopping-list-card.ts [hunk 1]
```diff
@@ -184,12 +184,12 @@ class HuiShoppingListCard
                         .itemId=${item.id}
                         @change=${this._completeItem}
                       ></ha-checkbox>
-                      <paper-input
-                        no-label-float
+                      <ha-textfield
+                        class="item"
                         .value=${item.name}
                         .itemId=${item.id}
                         @change=${this._saveEdit}
```
Comment (zsarnett) [inline src/panels/lovelace/cards/hui-shopping-list-card.ts:191]:
```suggestion
                        @input=${this._saveEdit}
```

File: src/panels/lovelace/entity-rows/hui-input-number-entity-row.ts [hunk 2]
```diff
@@ -108,23 +107,18 @@ class HuiInputNumberEntityRow extends LitElement implements LovelaceRow {
...
                   .disabled=${UNAVAILABLE_STATES.includes(stateObj.state)}
                   pattern="[0-9]+([\\.][0-9]+)?"
                   .step=${Number(stateObj.attributes.step)}
                   .min=${Number(stateObj.attributes.min)}
                   .max=${Number(stateObj.attributes.max)}
-                  .value=${Number(stateObj.state)}
+                  .value=${stateObj.state}
+                  .suffix=${stateObj.attributes.unit_of_measurement}
                   type="number"
                   @change=${this._selectedValueChanged}
```
Comment (zsarnett) [inline src/panels/lovelace/entity-rows/hui-input-number-entity-row.ts:119]:
```suggestion
                  @input=${this._selectedValueChanged}
```

File: src/panels/lovelace/cards/hui-shopping-list-card.ts (hunk 1 above)
Comment (bramkragten) [reply src/panels/lovelace/cards/hui-shopping-list-card.ts:191]: nope, we only want to update on blur

**Review by zsarnett — DISMISSED** [review]

File: src/panels/lovelace/cards/hui-shopping-list-card.ts [hunk 3]
```diff
@@ -29,6 +29,7 @@ import { SubscribeMixin } from "../../../mixins/subscribe-mixin";
 import { HomeAssistant } from "../../../types";
 import { LovelaceCard, LovelaceCardEditor } from "../types";
 import { SensorCardConfig, ShoppingListCardConfig } from "./types";
+import type { HaTextField } from "../../../components/ha-textfield";
```
[the remark this answers is not in the document]
Comment (zsarnett) [reply src/panels/lovelace/cards/hui-shopping-list-card.ts:32]: Whoops

**Review by zsarnett — APPROVED** [review]

**Pull request merged**